proto/ssn/type/candidate.proto

```diff
@@ -121,3 +121,13 @@ message PurchaseLineCandidate {
   // Unit price is the unit price of the line (with/without vat)
   string unit_price = 18;
 }
+message AnswerCandidate {
+  // Question asked in the request
+  string question = 1;
```

Comment:
I return question, answer pairs back so it is easy to identify them

Reply:
GPT API works as follows - with each request you have to send basically the whole context, which is an array of objects with 2 keys: role (either user or GPT) and the content (the actual message). The FE chat just renders messages based on the array order. See my [GPT POC](https://github.com/acornak/poc-gpt/tree/main).